### PR TITLE
Cleanup display of trust with transports

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM but you have whitespace issues

File: docs/source/markdown/podman-image-trust.1.md [hunk 1]
```diff
@@ -54,14 +56,17 @@ Trust may be updated using the command **podman image trust set** for an existin
             registry scope
     **reject**: do not accept images for this registry scope
 
-## show OPTIONS
-
-#### **--raw**
-  Output trust policy file as raw JSON
+#### show OPTIONS
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-image-trust.1.md:59]:
```suggestion
### show OPTIONS
```

Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman-image-trust.1.md:59]: Two was too big and conflicting with OPTIONS, but Four is too small and conflicts with the options themselves.  I'd go for in between with three hashes for this and `set OPTIONS`

File: docs/source/markdown/podman-image-trust.1.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ Trust may be updated using the command **podman image trust set** for an existin
 #### **--help**, **-h**
   Print usage statement.
 
+#### set OPTIONS
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-image-trust.1.md:43]:
```suggestion
### set OPTIONS
```

**Comment by TomSweeneyRedHat** [comment]
Couple of nits, plus the whitespace issues Matt noted.

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL
This looks like it is ready to go in.

**Comment by mheon** [comment]
You have an open comment from Tom

File: docs/source/markdown/podman-image-trust.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/source/markdown/podman-image-trust.1.md]: Fixed

File: docs/source/markdown/podman-image-trust.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-image-trust.1.md]: Fixed

File: docs/source/markdown/podman-image-trust.1.md [hunk 3]
```diff
@@ -54,14 +56,17 @@ Trust may be updated using the command **podman image trust set** for an existin
...
-
-#### **--raw**
-  Output trust policy file as raw JSON
+### show OPTIONS
 
 #### **--json**, **-j**
   Output trust as JSON for machine parsing
 
+#### **--noheading**, **-n**
+  Omit the table headings from the tust listings
```
Comment (umohnani8) [inline docs/source/markdown/podman-image-trust.1.md:65]: typo: s/tust/trust

**Comment by edsantiago** [comment]
Is it too late for a friendly suggestion? 
[pr13247.diff.txt](https://github.com/containers/podman/files/8119787/pr13247.diff.txt)

**Comment by rhatdan** [comment]
Applied

**Comment by edsantiago** [comment]
Thanks. LGTM.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**